pkgs/applications/audio/audacity/default.nix

```
@@ -29,24 +29,26 @@
 , soundtouch
 , pcre /*, portaudio - given up fighting their portaudio.patch */
 , linuxHeaders
-, at-spi2-core ? null
-, dbus ? null
-, epoxy ? null
-, libXdmcp ? null
-, libXtst ? null
-, libpthreadstubs ? null
-, libselinux ? null
-, libsepol ? null
-, libxkbcommon ? null
-, utillinux ? null
+, at-spi2-core
+, dbus
+, epoxy
+, libXdmcp
+, libXtst
+, libpthreadstubs
+, libselinux
+, libsepol
+, libxkbcommon
+, utillinux
```

> `anonymous function at <…>/pkgs/applications/audio/audacity/default.nix:1:1 called without required argument 'utillinux'`

https://github.com/NixOS/nixpkgs/blob/ce93c98ce2243a8228419fcfbe97fd87f3c5f99e/pkgs/top-level/aliases.nix#L817

```suggestion
, util-linux
```

same in `buildInputs`.